docs/notebooks/plot_slopes.ipynb

```diff
@@ -0,0 +1,2503 @@
+{
```

Comment:
<p>It would be good to change the label of the y-axis so it matches what you describe in the first paragraph. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/701/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>